compiler/src/dotty/tools/dotc/typer/Inferencing.scala

```diff
@@ -543,6 +543,11 @@ object Inferencing {
     case _ => tp
   }
 
+  def captureWildcardsCompat(tp: Type, pt: Type, compat: Compatibility)(using Context): Type =
```

Comment:
Why not leave `captureWildcardsCompat` in `Applications`? Then we don't need to pass the `Compatibility` argument.